### DISCUSSION TITLE
Decide how we want to move forward with Dashboard profile page

### DISCUSSION THREAD

**Discussion by npalaska** [distributed-system-analysis/pbench#3360] [Ideas]
Removal of the user management from the Pbench server has made the dashboard profile page merely a display page of the user's own information about email, first name, last name, and username. We can not provide edit options for these fields since we don't manage them anymore. Should we get rid of it or should we keep it for the display? 

Also account creation date is displayed on the profile page. How do we want to handle that? Do we see a value in displaying the user's first interaction with the Pbench server there? However, we don't have that mechanism to track the user creation date on the server yet but we could easily add that if we want to.

Open for discussion.

**Comment by dbutenhof** [comment]
Get rid of it unless we're going to add locally managed server attributes that could be displayed and edited, this really serves no purpose. (And right now, regardless of what we might choose to do in the future, I think we should dump it.)
